source/installguide/configuration.rst

```diff
@@ -1705,6 +1705,26 @@ add more servers to an existing zone.
    -  Path. The path to the zone's Secondary Staging Store.
 
 
+When a new Secondary Storage is added, the Management Server attempts to make
+existing templates available on the new Secondary Storage.
+
+CloudStack improves template availability using the configuration:
+
++----------------------------------------------+-------------------------------------------------------------------------------------------------------------+-----------+
+| Name                                         | Description                                                                                                 | Default   |
++==============================================+=============================================================================================================+===========+
+| copy.templates.from.other.secondary.storages | Allow templates to be copied from existing Secondary Storage servers (within the same zone or across zones) | true      |
+|                                              | when adding a new Secondary Storage, instead of downloading them from the source URL.                       |           |
++----------------------------------------------+-------------------------------------------------------------------------------------------------------------+-----------+
+
+This setting is enabled by default and can be configured globally or at zone level.
```

Comment:
when creating a zone it is enabled by default, so preventing it will be hard right? A race against the machine.